#174797507 Integrate TravisCI with readme badge

jpirumvaa opened the pull request:
### What does this PR do?
Configure TravisCI with a readme budge in a project

### Description of Task to be completed?

1. Pull the repo to get the project's latest version
2. Create a branch to work on
3. Add project on travis.org
4. Copy Markdown Budge and past it in ReadMe file
5. Setup .travis.yml file
6. Add, commit, and push changes to GitHub

### How should this be manually tested?
View on Github the README.md file. It must have` build `with value `passing`
### What are the relevant pivotal tracker stories?
[#174797507](https://www.pivotaltracker.com/n/projects/2465784/stories/174797507)

dushimeemma reviewed (commented):
use label please

mifeille commented on .travis.yml:
```diff
@@ -0,0 +1,10 @@
+  - npm install 
+  - npm test
+after_success:
+  - npm run coverage
```
Please add a newline here.

jpirumvaa committed:
chore(testing): add travisci finishes

-use travis as a continous integration tool
-add travisci badge in README.md file

[Finishes #174797507]
